src/lib/utility/logging/logging.h

```diff
@@ -5,32 +5,38 @@
 
 #include "utilityString.h"
 
+// Macro to strip the prefix from __FILE__
+#define RELATIVE_FILE_PATH (__FILE__ + SOURCE_PATH_PREFIX_LEN)
+
```

Comment:
constexpr is better

Reply:
Suppose we defined like this
```cpp
constexpr std::string_view relativeFilePath{__FILE__ + SOURCE_PATH_PREFIX_LEN};
```
Take in consideration that we already exposed **Sourcetrail_utility_logging** [here](https://github.com/OpenSourceSourceTrail/Sourcetrail/pull/10/files#diff-7567fde99fcba090ecb1958dd4ab9ba4367c10ba5d444e580cb61c3b404ced3aR413)

So **relativeFilePath** will always be **src/lib/utility/logging/logging.h**

Remove **constexpr**? Why I need to expose a new variable?

I see that the current implementation is **fit** to our functionality.